### ISSUE TITLE
exportData modifier fails on selected: true

### ISSUE THREAD

**Issue by bmcfee** [DataTables/Select#6]
I'm seeing the following behavior on Select 1.0.1, Buttons 1.0.3, DataTables 1.10.9.  I have a table of 300 rows with 125 selected.

``` javascript
> table.buttons.exportData({modifier: {selected: undefined}})
< Object {header: Array[13], footer: null, body: Array[300]}
> table.buttons.exportData({modifier: {selected: false}})
< Object {header: Array[13], footer: null, body: Array[175]}
> table.buttons.exportData({modifier: {selected: true}})
< Object {header: Array[13], footer: null, body: Array[0]}
```

As you can see, the `selected: true` filter seems broken, as it gives an empty body instead of `Array[125]`.

This of course causes some problems down the road when I want to export-to-csv just the selected rows.

**Comment by DataTables** [comment]
Ca you link to a test case showing the issue please?

**Comment by bmcfee** [comment]
Additional note: I'm not the only one to hit this issue.  [This thread](https://datatables.net/forums/discussion/29956/copying-or-printing-selected-rows-doesn-t-export-any-rows) also discusses it, and the proposed solution appears to work for me.

**Comment by DataTables** [comment]
Thanks for the link to the thread. This issue was fixed by [this commit is Buttons](https://github.com/DataTables/Buttons/commit/75d79487b60613eaea3b72f29058dc6565bf923b). It will be included in the 1.0.4 release and is in the nightly version if you want to use it there.

**Comment by bmcfee** [comment]
Fantastic.  Thanks!
